### PR TITLE
MBS-9936: Move from CJS style to ES6 (imports, exports) (VI)

### PR DESCRIPTION
**Pull request by reosarevok**
scripts/edit

I would love to eslint --fix all of these too, but let's do one thing at a time so it's reviewable.

### CODE REVIEW

File: root/static/scripts/edit/MB/Control/Area.js [hunk 1]
```diff
@@ -29,4 +29,4 @@ MB.Control.Area = function () {
     });
 };
 
-module.exports = MB.Control;
+export default MB.Control;
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/Control/Area.js]: There's an import in root/static/scripts/place.js which this breaks.

Perhaps replace this line with something like `export const initializeArea = MB.Control.Area;` and change the import to `import {initializeArea} from './edit/MB/Control/Area';`.

File: root/static/scripts/edit/MB/Control/Bubble.js [hunk 2]
```diff
@@ -332,4 +332,4 @@ MB.Control.initializeBubble = function (bubble, control, vm, canBeShown) {
     return bubbleDoc;
 };
 
-module.exports = MB.Control;
+export default MB.Control;
```
Comment (mwiencek) [inline root/static/scripts/edit/MB/Control/Bubble.js]: This breaks an import in root/static/scripts/place.js. I suggest `export const initializeBubble = MB.Control.initializeBubble;` and no default export.

File: root/static/scripts/work.js [hunk 3]
```diff
@@ -19,8 +19,8 @@ const {lp_attributes} = require('./common/i18n/attributes');
 const MB = require('./common/MB');
 const scriptArgs = require('./common/utility/getScriptArgs')();
 const {Lens, prop, index, set, compose3} = require('./common/utility/lens');
-const {buildOptionsTree} = require('./edit/forms');
-const {initializeBubble} = require('./edit/MB/Control/Bubble');
+import forms from './edit/forms';
+import bubble from './edit/MB/Control/Bubble';
```
Comment (mwiencek) [inline root/static/scripts/work.js]: Per comment above,
```suggestion
import {initializeBubble} from './edit/MB/Control/Bubble';
```

File: root/static/scripts/edit/forms.js [hunk 4]
```diff
@@ -279,4 +279,4 @@ ko.bindingHandlers.withLabel = {
     }
 };
 
-module.exports = MB.forms;
+export default MB.forms;
```
Comment (mwiencek) [inline root/static/scripts/edit/forms.js]: I'd prefer no default export here too, and adding named exports for `buildOptionsTree`, `linkTypeOptions`, etc., which is how this module was previously consumed.

`MB.forms` is a bogus namespace from before we had actual modules - given that this module holds several utility functions and doesn't have/need a primary export, it makes much more sense to use named exports.

File: root/static/scripts/work.js [hunk 5]
```diff
@@ -19,8 +19,8 @@ const {lp_attributes} = require('./common/i18n/attributes');
 const MB = require('./common/MB');
 const scriptArgs = require('./common/utility/getScriptArgs')();
 const {Lens, prop, index, set, compose3} = require('./common/utility/lens');
-const {buildOptionsTree} = require('./edit/forms');
-const {initializeBubble} = require('./edit/MB/Control/Bubble');
+import forms from './edit/forms';
```
Comment (mwiencek) [inline root/static/scripts/work.js]: Per comment above,
```suggestion
import {buildOptionsTree} from './edit/forms';
```

File: root/static/scripts/edit/forms.js (hunk 4 above)
Comment (reosarevok) [reply root/static/scripts/edit/forms.js]: Do I just *remove* MB.forms completely then, leaving only separate functions?

Comment (mwiencek) [reply root/static/scripts/edit/forms.js]: It's used in root/components/relationship-editor.tt by knockout so I wouldn't remove the reference, just the export for it.

File: root/static/scripts/place.js [hunk 6]
```diff
@@ -10,7 +10,7 @@ const _ = require('lodash');
 
 const isBlank = require('./common/utility/isBlank');
 const initializeDuplicateChecker = require('./edit/check-duplicates');
```
Comment (mwiencek) [inline root/static/scripts/place.js]:
```suggestion
const initializeDuplicateChecker = require('./edit/check-duplicates').default;
```
There's a similar issue in root/static/scripts/series.js

**Pull request merged**